Makefile

```diff
@@ -33,8 +33,8 @@ ifneq ($(shell id -u), 0)
 	@echo "You must be root to perform this action."
 	exit 1
 endif
-	cd /tmp && curl https://get.mocaccino.org/luet/get_luet_root.sh | sh
-	cd /tmp && luet install -y extension/makeiso
+	cd /tmp && curl https://raw.githubusercontent.com/rancher-sandbox/cOS-toolkit/master/scripts/get_luet.sh | sh
```

Comment:
Also switching to our repositories to consume deps. It's way more faster as everything is in one repo (and we are sure we build against the same versions)